docs/reference/overview_of_strategies.rst

```diff
@@ -536,7 +536,7 @@ elsewhere as HardProber. The strategy starts by playing D, D, C, C on the first
 four rounds and then defects forever if the opponent cooperates on rounds
 two and three. Otherwise Prober plays as TitForTat would.
 
```

Comment:
NOTE: This is a possible minor typo completely unrelated to this PR.  It didn't seem worth raising a separate issue. (2 of 2)